langchain/utilities/powerbi.py

```diff
@@ -16,6 +16,15 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+try:
```

Comment:
not super familiar with python import execution, will this only be run if someone tries importing from langchain.utilities.powerbi or would it also be run if someone tried importing something unrelated from langchain.utilities (which imports from langchain.utilities.powerbi)